geospaas_harvesting/harvest.py

```diff
@@ -222,7 +222,7 @@ def main():
         sys.exit(1)
     LOGGER.info('Updating vocabularies...')
 
-    update_vocabularies.Command().handle() #updating the vocabulary with this command
+    #update_vocabularies.Command().handle() #updating the vocabulary with this command
```

Comment:
?

Reply:
that is for my debug! wrongly pushed! sorry. thanks